src/modules/site-v2/templates/tools/heritability_calculator/result.html

```diff
@@ -48,7 +48,7 @@
     
       {% if hr.status == JobStatus.ERROR %}
       {% if session["is_admin"] %}
-      <p><strong>Error:</strong> <span class="text-danger">{{ operation.error }}</span></p>
+      <p><strong>Error:</strong> <span class="text-danger">(code {{ error.code }}) {{ error.message }}</span></p>
```

Comment:
The `JobPipeline` returns the error with a code & message, so this displays them more neatly